doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md

```diff
@@ -123,3 +123,29 @@ Settings for outbound HTTP connectivity of Pimcore - needed e.g. for Pimcore Upd
 Possibility for configuring different newsletter delivery settings from the default e-mail settings.
  
  
+## Access system config in PHP Controller
+Using \Pimcore\Config::getSystemConfig(); is deprecated. Access the system config using the default symfony way in your controllers:
+```php 
+<?php
+
+namespace AppBundle\Controller;
+
+use Pimcore\Controller\FrontendController;
+use Symfony\Component\HttpFoundation\Request;
+
+class DefaultController extends FrontendController
+{
+ 
+    /**
+     * @param Request $request
+     */
+    public function defaultAction(Request $request)
+    {
+    
+        $pimcoreconfig = $this->getParameter('pimcore.config');
```

Comment:
type-hinting config class should be the recommended way. please see https://github.com/pimcore/pimcore/blob/ea7c3906e85a71af0138b67ee8b242b3a29599f2/bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php#L313

Reply:
Both ways are fine, but using the config service is probably a bit more handy 😉

Reply:
I guess we should just include both ways in the docs, then it's up to the dev 😊

Reply:
@Cruiser13 could you please update your PR and include the alternative way described by @dvesh3 
Thanks a lot!

Reply:
Sorry, I didn't see your message. Thanks for taking care of the docs!